website/management/commands/check_trademarks.py

```
@@ -12,27 +12,53 @@
     }
-    response = requests.post(url, data=payload, headers=headers)
-    if response.status_code == 200:
-        return response.json()
-    else:
-        print(f"Error: Received status code {response.status_code} - {response.reason}")
-        print(response.json())
-    return None
+
+    try:
+        initial_payload = {"keywords": f'["{term}"]', "start_index": "0"}
+        response = requests.post(url, data=initial_payload, headers=headers)
```

Ruff (0.14.4)

Probable use of `requests` call without timeout

(S113)